app/commands/tooling_job/cancel.rb

```diff
@@ -0,0 +1,62 @@
+module ToolingJob
+  class Cancel
+    include Mandate
+
+    initialize_with :iteration_uuid, :type
+
+    def call
+      iteration = Iteration.find_by!(uuid: iteration_uuid)
+
+      items = client.query({
+                             table_name: Exercism.config.dynamodb_tooling_jobs_table,
+                             index_name: "iteration_type",
+                             expression_attribute_values: {
+                               ":IU" => iteration_uuid,
+                               ":TP" => type,
+                               ":JS" => :pending
+                             },
+                             expression_attribute_names: {
+                               "#IU": "iteration_uuid",
+                               "#TP": "type",
+                               "#ID": "id",
+                               "#JS": "job_status"
+                             },
+                             key_condition_expression: "#IU = :IU AND #TP = :TP",
+                             filter_expression: "#JS = :JS",
+                             projection_expression: "#ID"
+                           }).items
```

Comment:
If you drop the first `{` onto a new line I think this will indent less severely.